addons/website_blog/views/website_blog_components.xml

```diff
@@ -0,0 +1,319 @@
+============================================================================ -->
+<template id="website_blog.date_selector">
+    <select name="archive" oninput="location = this.value;" class="custom-select">
+        <option t-att-value="blog_url(date_begin=False, date_end=False) if blog else '/blog'"
+                t-att="[('selected' if (not date_begin) else 'unselected' ) , 'true' ]">
+                -- All dates
+        </option>
+
+        <optgroup t-foreach="nav_list" t-as="year" t-attf-label="#{year}">
+            <option t-foreach="nav_list[year]" t-as="months"
+                    t-att="[('selected' if date_begin and (months['date_begin'] == date_begin) else 'unselected' ) , 'true' ]"
+                    t-attf-value="#{blog_url(date_begin=months['date_begin'], date_end=months['date_end'])}">
+                <t t-esc="months['month']"/>
+                <t t-esc="year"/>
+            </option>
+        </optgroup>
+    </select>
+</template>
+
+<!-- ======   Template: Post Author   ==========================================
+============================================================================ -->
+<template id="website_blog.post_author">
+    <div t-attf-class="o_not_editable align-items-center position-relative #{additionnal_classes or ''}">
+        <div t-if="blog_post.author_avatar"
+             t-field="blog_post.author_avatar"
+             style="line-height:1"
+             t-options='{"widget": "image", "class": "rounded-circle " + "o_wblog_author_avatar mr-1" if hide_date else  "o_wblog_author_avatar_date mr-2"}' />
+        <div t-att-class="'small font-weight-bold' if not hide_date else ''" style="line-height:1">
+            <span t-field="blog_post.author_id" t-options='{ "widget": "contact", "fields": ["name"]}'/>
+            <small t-if="not hide_date" t-field="blog_post.post_date" t-options='{"format": "d MMMM, yyyy"}'/>
+        </div>
+    </div>
+</template>
+
+<!-- ======   Template: Post Breadcrumbs   =====================================
+============================================================================ -->
+<template id="website_blog.post_breadcrumbs">
+    <nav aria-label="breadcrumb" t-attf-class="breadcrumb flex-nowrap py-0 px-0 css_editable_mode_hidden #{additionnal_classes or ''}">
+        <li t-if="blogs" class="breadcrumb-item"><a href="/blog">All Blogs</a></li>
+        <li class="breadcrumb-item position-relative">
+            <a t-if="not blogs" t-attf-href="#{blog_url(tag=None, date_begin=None, date_end=None)}" t-esc="blog.name"/>
+            <t t-else="">
+                <a t-attf-href="#{blog_url(tag=None, date_begin=None, date_end=None)}" t-esc="blog.name"/>
+                <a type="button" class="dropdown-toggle dropdown-toggle-split pr-0 pl-1" href="#" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"/>
+
+                <div class="dropdown-menu dropdown-menu-right">
```

Comment:
I think we decided with @sbuhl and @JKE-be to remove that feature. It can make sense to have an "advanced" breadcrumb in some system (like a file system) but giving the possibility to switch blog from one blog post does not feel useful and complexify the code and the UI for no good reason to us (especially when clicking on the normal link would go back to the same beautiful page where you can switch blog anyway).

Reply:
Agree with QSM, not needed.

Reply:
Yep, I had mixed feedbacks about this and I proposed @sbuhl to let me iterate more to find a better solution (and than I forgot sorry ^^ ). I'll remove it 👍

Reply:
I have already removed in my local diff.